README.md

```diff
@@ -481,6 +507,25 @@ neoForge {
 In addition, you can add additional data-files to the `interfaceInjectionData` configuration using normal
 Project dependency syntax in your dependencies block.
 
+#### Publication of Interface Injection Data
+The publication of interface injection data follows the same principles as the publication of access transformers.
+
+The published file name is `interfaceinjection.json` for a single file,
+or `interfaceinjection1.json` and so on for multiple files.
```

Comment:
This should be documented consistently with the ATs section.

Reply:
Happy?